apps/frontend/src/modules/surveys/components/studio/distribute/sharable-links/QRCodeview.tsx

```diff
@@ -48,12 +48,12 @@ export default function QRCodeView({ url, name }: { url: string; name: string })
     };
 
     return (
-        <div className="flex w-[778px] flex-col items-center gap-8 pt-8">
+        <div className="flex w-[600px] flex-col items-center gap-8 pt-8">
```

Comment:
@afolasope I thing this width should be based on screen size.

Reply:
So, for this container, we can add `lg:w-[778px]` to the `classNames` and for the `QRCode` component, we can can add `size-[210px] lg:size-[397px]` to the `classNames`